sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -253,7 +256,7 @@ template <typename PropertyT>
 struct IsRuntimeProperty
     : std::bool_constant<
           std::is_base_of_v<property_key_base_tag, PropertyT> &&
-          !std::is_base_of_v<compile_time_property_key_base_tag, PropertyT>> {};
+          std::is_base_of_v<run_time_property_key_base_tag, PropertyT>> {};
```

Comment:
Why is this needed and why isn't it a separate PR?

Reply:
I would really love to delay this until our ongoing properties discussion finishes... @gmlueck , @Pennycook , @rolandschulz